scripts/qa_crowbarsetup.sh

```diff
@@ -2047,6 +2047,8 @@ function reboot_nodes_via_ipmi
                 sleep $((50 + RANDOM % 20))
 
                 ipmitool -H $ip -U root -P $pw chassis bootdev pxe
+                # Please do not touch this sleep. It is important for the ipmi to work on qa hardware
+                sleep $((50 + RANDOM % 20))
```

Comment:
I agree with @jdsn and @aplanas. The need for the sleep should be explained, but even better, if it was using `wait_for` then it would be self-explanatory and also a lot more reliable than sleeping for a random amount of time.